server/controllers/boardController.js

```diff
@@ -19,5 +19,28 @@ boardController.getBoards = (req, res, next) => {
       });
     });
 };
+boardController.getBoard = (req, res, next) => {
+  console.log('running boardController.getBoard. req.param: ', req.params)
+  const { board_id } = req.params;
+
+  Board.findOne({_id: board_id })
+    .then(response => {
+      res.locals.board = response;
+      return next();
+    })
```

Comment:
`findById()` maybe better: https://mongoosejs.com/docs/api/model.html#model_Model-findById

from the docs:
```
[findOne() and findbyId()] treat undefined [differently]. If you use findOne(), you'll see that findOne(undefined) and findOne({ _id: undefined }) are equivalent to findOne({}) and return arbitrary documents. However, mongoose translates findById(undefined) into findOne({ _id: null }).
```

This may protect us against accidentally returning all the entries because we are setting the value of `_id` to `undefined`

```suggestion
  Board.findbyId(board_id)
    .then(response => {
      res.locals.board = response;
      return next();
    })
```

Reply:
Agreed!

Reply:
implemented and functioning